xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd

```diff
@@ -550,4 +551,82 @@ Rail transport, Roads and Road transport
 		</xsd:complexContent>
 	</xsd:complexType>
 	<!-- ======================================================================= -->
+	<xsd:element name="VehicleJourneyStopAssignment" abstract="false" substitutionGroup="StopAssignment">
+		<xsd:annotation>
+			<xsd:documentation>Change to a PASSENGER STOP ASSIGNMENT for a specific VEHICLE JOURNEY +v1.1</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:restriction base="VehicleJourneyStopAssignment_VersionStructure">
+					<xsd:sequence>
+						<xsd:sequence>
+							<xsd:sequence>
+								<xsd:group ref="EntityInVersionGroup" minOccurs="0"/>
+							</xsd:sequence>
+							<xsd:sequence>
+								<xsd:group ref="DataManagedObjectGroup"/>
+							</xsd:sequence>
+							<xsd:sequence>
+								<xsd:group ref="AssignmentGroup"/>
+							</xsd:sequence>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="StopAssignmentGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="PassengerStopAssignmentGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="VehicleJourneyStopAssignmentGroup"/>
+						</xsd:sequence>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="VehicleJourneyStopAssignmentIdType"/>
+				</xsd:restriction>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:complexType name="VehicleJourneyStopAssignment_VersionStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for DYNAMIC PASSENGER STOP ASSIGNMENT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="PassengerStopAssignment_VersionStructure">
+				<xsd:sequence>
+					<xsd:group ref="VehicleJourneyStopAssignmentGroup">
+						<xsd:annotation>
+							<xsd:documentation>Elements for a VEHICLE JOURNEY STOP ASSIGNMENT.  Original Passenger STop Assignment for which this is a change.</xsd:documentation>
+						</xsd:annotation>
+					</xsd:group>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="VehicleJourneyStopAssignmentGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for a VEHICLE JOURNEY STOP ASSIGNMENT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element ref="VehicleJourneyRef" minOccurs="0" maxOccurs="unbounded"/>
+			<xsd:element ref="PassengerStopAssignmentRef" minOccurs="0"/>
+		</xsd:sequence>
+	</xsd:group>
+	
+	<xsd:complexType name="VehicleJourneyStopAssignmentsInFrame_RelStructure">
```

Comment:
It seems that most of this change is already in the current master. But VehicleJourneyStopAssignmentsInFrame_RelStructure is not.

Reply:
So what is the best way to go from here: shall we close this one and creat a new PR or will Merge be able to manage this ? (we obviusly need to be able to carry this info in a Frame)

Reply:
The foremost problem is there is zero documentation other than mentioning of the change request.

Reply:
Right... I just added the "needs documentation update" label